mako_config/mobile-bert.config

```diff
@@ -0,0 +1,60 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file is a protobuff text format representation of a Mako
+# BenchmarkInfo message.
+#
+# For more information about text format protobuffs see: go/proto-text-format
```

Comment:
go links are not public. This is the best public reference I can find https://developers.google.com/protocol-buffers/docs/reference/cpp/google.protobuf.text_format

(Really??!)